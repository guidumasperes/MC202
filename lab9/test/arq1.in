500 15
1 C-137 10
1 B-222 31
2 B-222
2 C-137
3 B-222
2 B-222
2 C-137
9
