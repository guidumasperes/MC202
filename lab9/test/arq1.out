31
10
chave removida!
chave nao encontrada!
10
